Makefile

```diff
@@ -146,12 +146,13 @@ delete-integration-test-dev-cluster: build ## Delete the test cluster for use wh
 ##@ Code Generation
 
 .PHONY: generate-always
-generate-always: ## Generate code (required for every build)
+generate-always:  pkg/ami/static_resolver_ami.go pkg/nodebootstrap/maxpods.go pkg/addons/default/assets/aws-node.yaml
```

Comment:
these make targets are not  `.PHONY`, and all these files are available in repo, hence I am struggling to find out in which cases these target will be triggered. Can you help to clarify a little bit ?

Reply:
(testing the comment from golang)

Reply:
We could do that as well. However, do you feel that we need to run some targets (e.g. _update-ami_ or _update-maxpods_, etc) in CI ? Could be in PR or master ? or before release ?

Reply:
No, I don't want them to run in CI at this point. That's why I'm moving them out of the main building path. I don't want to block PRs on updating external files. That should go into its own PR and should be a conscious decision.

It would be ok, though, to have CI check if these need updating periodically and open a PR, I just don't want development slowed down when an unrelated external dependency needs updating.

Reply:
Oh i got it. LGTM :100: